### PR TITLE
Pro alanase

### PR DESCRIPTION
**Pull request by Alexander-Sol**
New Site-Specific Endoprotease that Targets Proline and Alanine

    Cleaves primarily C-terminal to proline and alanine residues
    Active at acidic pH (1–5.5) with a pH optimum of ~1.5
    Short digestion times of 1–2 hours

### CODE REVIEW

File: mzLib/Proteomics/ProteolyticDigestion/proteases.tsv [hunk 1]
```diff
@@ -22,3 +22,4 @@ collagenase	GPX|GPX			full
 StcE-trypsin	"TX|T,TX|S,SX|T,SX|S,K|,R|"			full		StcE/Trpsin			
 CNBr_old	M|			full	MS:1001307	CNBr	(?<=M)		
 CNBr_N	|M			full	MS:1001307	CNBr	(?<=M)	Test on M	
+ProAlanase	"P|,A"			full
```
Comment (trishorts) [inline mzLib/Proteomics/ProteolyticDigestion/proteases.tsv:25]: should this be "P|,A|"

I can't remember. It's c-terminal to both.

**Comment by trishorts** [comment]
you'll have to update your branch
git fetch upstream
git merge upstream/master
git push

**Comment by Alexander-Sol** [comment]
>

**Comment by Alexander-Sol** [comment]
Not sure why this broke ~150 tests. Going to create a new branch

**Pull request closed without merging**